ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -83,6 +83,10 @@
             // before initializing the form container load all the locale specific json resources
             // for runtime
             const formLanguage = formContainer.getLang();
+            if (formLanguage) {
```

Comment:
If formLanguage is not a valid language tag, this may lead to unexpected behavior. Can you add error handling to ensure that formLanguage is a valid input before attempting to set the text direction.